src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Test/ScenarioTests/AsTests.cs

```diff
@@ -39,6 +39,13 @@ public void TestAnalysisServicesServerScaleUpDown()
         }
 
         [Fact]
+        [Trait(Category.AcceptanceType, Category.LiveOnly)]
```

Comment:
@taiwu any reason this test is `LiveOnly`? We want to make sure all tests are `CheckIn` so they are run during the regular CI job